sass/_community.scss

```
@@ -0,0 +1,454 @@
+// Community landing page (templates/community-home.html)
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)